tests/Feature/MovieCreateTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MovieCreateTest extends TestCase
+{
+    use RefreshDatabase;
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_movie_create_screen_requires_authorized_user()
+    {
+        $user = User::factory()->make();
+
+        $this->actingAs($user)
+        ->get('/movies')
+        ->assertStatus(200);
+    }
+
+    public function test_movie_create_new_movie_can_be_added() // No fails
+    {
+        $user = User::factory()->make();
+
+        $this->actingAs($user)
+        ->post('/movies', [
+            'title' => 'A Test movie',
+            'release_date' => '06/12/2031',
+            'poster' => 'https://m.media-amazon.com/images/M/MV5BMWZmYTI4MDctMzU4OC00ODJmLTkwMTgtYjRmMDRkMzc3NWZkXkEyXkFqcGdeQXVyNTAyODkwOQ@@._V1_.jpg',
+            'trailer' => 'https://www.youtube.com/watch?v=-1dSY6ZuXEY',
+            'blurb' => 'This is a sample blurb for the movie A Test movie'
+        ])->assertSessionHasNoErrors()
+        ->assertRedirect();
+    }
+
+    public function test_movie_create_invalid_data_throws_error()
```

Comment:
Looks okay :D Just one suggestion on the naming of the test. 

This should literally discribe what you are testing, so if you are testing  that the app throws validation error when you have invalid poster value. Your test should be named something like test_invalid_poster_movie_attribute_throws_validation_error.

Reply:
Name for various tests have been improved to more concisely and literally describe test